File: src/main/java/nu/fgv/register/server/repository/SpexRepository.java
package nu.fgv.register.server.repository;

import nu.fgv.register.server.model.Spex;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.querydsl.QuerydslPredicateExecutor;
import org.springframework.graphql.data.GraphQlRepository;

@GraphQlRepository
public interface SpexRepository extends JpaRepository<Spex, Long>, QuerydslPredicateExecutor<Spex> {
}

File: src/main/kotlin/nu/fgv/register/server/repository/SpexRepository.kt
package nu.fgv.register.server.repository

import nu.fgv.register.server.domain.Spex
import org.springframework.data.repository.PagingAndSortingRepository
import org.springframework.data.rest.core.annotation.RepositoryRestResource

@RepositoryRestResource
interface SpexRepository : PagingAndSortingRepository<Spex, Long>
